README.md

```diff
@@ -1,21 +1,29 @@
 <p align="center"><a href="https://missingpersonsug.org/" target="_blank"><img src="public/media/image_of_person.jpg" width="400" alt="Missing Person in Uganda"></a></p>
 
 ## Missing persons in Uganda Backend
-This is a backend for the Missing persons UG which mirrors the official data file at https://github.com/wkambale/missingpersons/blob/main/data.json
+
+This is a backend for the Missing persons UG which mirrors the official data file
```

Comment:
I think it is worth noting that the data file is nolonger the source of truth.

Reply:
@skapeyi I am going to ask @wkambale as the project lead to actually make that change 

He is the one with the authority when git blame is checked 

Over to you Wes

Reply:
hey folks, i actually don't think i follow here.

@skapeyi, what you mean the data file is no longer the source of truth anymore? some context.